src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AdjudicationsResourceTest.java

```diff
@@ -399,4 +400,149 @@ public void returns403IfInvalidRole() {
             assertThatStatus(response, 403);
         }
     }
+
+    @Nested
+    public class AdjudicationHearings {
+
+        final String token = validToken(List.of("ROLE_MAINTAIN_ADJUDICATIONS"));
+        final String invalidToken = validToken(List.of("ROLE_SYSTEM_USER"));
+        final HttpEntity<?> validRequest = createHttpEntity(token, Map.of("dateTimeOfHearing","2022-10-24T10:12:44", "hearingLocationId", "-31", "oicHearingType", "GOV_ADULT"));
+        final HttpEntity<?> invalidRequest = createHttpEntity(token, Map.of("dateTimeOfHearing","not a date time", "hearingLocationId", "-31", "oicHearingType", "GOV_ADULT"));
+        final HttpEntity<?> invalidLocationRequest = createHttpEntity(token, Map.of("dateTimeOfHearing","2022-10-24T10:12:44", "hearingLocationId", "1", "oicHearingType", "GOV_ADULT"));
+        final HttpEntity<?> invalidRoleRequest = createHttpEntity(invalidToken, Map.of("dateTimeOfHearing","2022-10-24T10:12:44", "hearingLocationId", "1", "oicHearingType", "GOV_ADULT"));
+        final HttpEntity<?> invalidTypeRequest = createHttpEntity(token, Map.of("dateTimeOfHearing","2022-10-24T10:12:44", "hearingLocationId", "-31", "oicHearingType", "WRONG"));
+
+
+        @Test
+        public void createHearingReturns403 () {
```

Comment:
You could add a bit more context about what you're testing to some of the method names in this class? Perhaps `createHearingReturns403ForInvalidRole`, etc